app/views/layouts/users.html.slim

```diff
@@ -0,0 +1,14 @@
+doctype html
```

Comment:
How about name it as `auth.html.slim` instead of `users.html.slim` as the layout is for generic purpose, an `app layout` or an `auth layout` 🤔

Reply:
totally agree.